graphistry/database_clients/SpannerGraph.py

```diff
@@ -0,0 +1,287 @@
+import os
+import pandas as pd
+import json
+import time
+import logging
+from typing import Any
+from google.cloud.spanner_dbapi.connection import connect
+from google.cloud.spanner_v1.data_types import JsonObject
+from google.cloud.spanner_v1 import KeySet
```

Comment:
this can be an optional dep in setup.py: https://github.com/graphistry/pygraphistry/blob/7f5b2db8f54e3763dbe8a67793c5d707f935626b/setup.py#L48

moving to dynamic imports may help, not sure